flutter/cpp/mlperf_driver.h

```diff
@@ -56,13 +56,6 @@ class MlperfDriver : public ::mlperf::SystemUnderTest {
   // Flush the staged queries immediately.
   void FlushQueries() override { backend_->FlushQueries(); }
 
-  // Called by loadgen to show us the recorded latencies.
-  void ReportLatencyResults(
-      const std::vector<::mlperf::QuerySampleLatency>& latencies_ns) final {
-    latencies_ns_.insert(latencies_ns_.end(), latencies_ns.begin(),
```

Comment:
`latencies_ns_` is used to calculate performance metric:
https://github.com/mlcommons/mobile_app_open/blob/c7dac5e560674652b28a1ad768099211c3cb105d/flutter/cpp/mlperf_driver.h#L67-L68
When it is empty, app UI gets `0.0` latency for all results, which results in `infinity` performance value, and this causes the error that we can see on the screenshot.

We can either create a patch for loadgen to revert the commit that removes `ReportLatencyResults`, or we can parse loadgen summary logs.

It would be ideal if loadgen exported summary in its API instead of only printing it to file but I guess we won't get this anytime soon.

Reply:
Thanks, I created a patch to revert LoadGen's removal. It works.